client/src/templates/Challenges/projects/frontend/Show.js

```diff
@@ -176,7 +176,7 @@ export class Project extends Component {
                 <br />
                 <Spacer />
               </Col>
-              <CompletionModal blockName={blockName} />
+              <CompletionModal block={block} superBlock={superBlock} />
```

Comment:
```suggestion
              <CompletionModal
                block={block}
                blockName={blockName}
                superBlock={superBlock}
              />
```
Currently `CompletionModal` still needs `blockName`.  I think it's worth refactoring it so it doesn't, but I can handle that in another PR.